lighthouse-cli/printer.ts

```diff
@@ -110,7 +110,13 @@ function createOutput(results: Results, outputMode: OutputMode): string {
         if (auditResult.comingSoon === true)
           return;
 
-        let lineItem = ` ${log.doubleLightHorizontal} ${formatAggregationResultItem(auditResult.score)} ${auditResult.description}`;
+        let lineItem = ` ${log.doubleLightHorizontal}`;
+        if (auditResult.error) {
+          lineItem += ` ${log.redify('-')}`;
```

Comment:
![screen shot 2017-02-01 at 15 28 35](https://cloud.githubusercontent.com/assets/316891/22530965/201a0ba6-e893-11e6-9b3e-aa1f5a88ce8f.png)

this is kind of weak, so feel free to suggest a different character :)